CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # CHANGELOG
 
-## Next
+## 5.2.0
 
-* Your contribution here.
+* [#166](https://github.com/digitalplaywright/mongoid-slug/issues/166): Feature to configure slug builder globally
```

Comment:
Match the format of the text elsewhere. Should have `- [@name]().` at the end. Also maybe just say "Configure slug build globally", the "Feature to" is redundant.